src/common/k8s-api/json-api.ts

```diff
@@ -194,11 +205,12 @@ export class JsonApi<D = JsonApiData, P extends JsonApiParams = JsonApiParams> {
   protected writeLog(log: JsonApiLog) {
     if (!this.config.debug) return;
     const { method, reqUrl, ...params } = log;
-    let textStyle = "font-weight: bold;";
+    // let textStyle = "font-weight: bold;";
 
-    if (params.data) textStyle += "background: green; color: white;";
-    if (params.error) textStyle += "background: red; color: white;";
-    console.log(`%c${method} ${reqUrl}`, textStyle, params);
+    // if (params.data) textStyle += "background: green; color: white;";
+    // if (params.error) textStyle += "background: red; color: white;";
+    //console.log(`%c${method} ${reqUrl}`, textStyle, params);
+    logger.info(`[JSON-API] request ${method} ${reqUrl}`, params);
```

Comment:
Will cleanup this after we have decided right logging level. Would `debug` be better?

Reply:
probably, log lines we are getting now are a bit too verbose for `info` level.